Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node
+ARG SERVICE_NAME
+
+# These steps ensure that npm install is only run when package.json changes
+RUN mkdir -p /usr/src/$SERVICE_NAME
+ADD ./package.json /usr/src/$SERVICE_NAME/package.json
+WORKDIR /usr/src/$SERVICE_NAME
+RUN npm install --production
+
+ADD . /usr/src/$SERVICE_NAME
```

Comment:
For now it might be easier to do so because exosphere should just take services with dockerfiles and deploy them.

Eventually though we could have exosphere-sdk generate these automatically on `exo setup`